src/backend/joanie/tests/core/test_api_contract_definitions.py

```diff
@@ -144,8 +144,9 @@ def test_api_contract_definition_preview_template_success(
             r"This document certifies that the student wants to enroll to the course",
         )
         self.assertRegex(document_text, r"Student's signature")
-        self.assertRegex(document_text, r"Representative's signature")
         self.assertRegex(document_text, r"[SignatureField#1]")
+        self.assertRegex(document_text, r"Representative's signature")
```

Comment:
If you try to check if this string is present in another one, using `assertIn` is probably faster ?

Reply:
Didn't really think about it, many assert were already built like this in this file.

Reply:
I know you can do it.